Add access review error (#13413)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accessreviewerror.md

---
title: "accessReviewError resource type"

> description: "Represents error happened within access review instance lifecycle."

Reviewer: ```suggestion
description: "Represents an error that occurred within an access review instance lifecycle."
```

author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

# accessReviewError resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> [!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]

Reviewer 2: The disclaimer text includes the phrase "in this section", which does not make sense in the context of this topic because it is not part of a section in the TOC. Can you revise the disclaimer text in the include file to be more clear instead of relying on a location to convey which API is not deprecated? Maybe the text should just say "This API". I would change "older" version to "previous" version, and state that it is deprecated, not "will be" deprecated since we've already marked it as deprecated.

Author: @isabelleatmsft  Hi Isabelle, can't fine the file accessreviews-disclaimer-v2.md - was it deprecated ? could you suggest what the change should be here?

Reviewer 3: @Lauragra i made a PR for those changes: https://github.com/microsoftgraph/microsoft-graph-docs/pull/13569

> Represent errors happen in review instance lifecycle. **code** represent error type. **message** contain error details. This resource is read-only.

Reviewer: ```suggestion
Represents an error that occurred in an access review instance lifecycle. This resource is read-only.
```

Inherits from [genericError](../resources/genericerror.md).

## Properties
| Property                     | Type     | Description                          |
| :--------------------------- | :------  | :----------                          |

> | code  |String  | Inherit from microsoft.graph.genericError. Error happened in review life cycle.|
> | message |String | Inherit from microsoft.graph.genericError. Error details.|

Reviewer: Let's link to the **genericError** resource.
```suggestion
| code  |String  | Represents the error type. Inherited from genericError. Read-only. |
| message |String | Represents the error details. Inherited from genericError. Read-only.|
```
